frontend/src/components/map/LeafletMapContainer.tsx

```diff
@@ -45,7 +46,7 @@ const LeafletMapContainer = ({ originId, destinationId, locationIds, showLines,
       polygons.push(
         <Polygon
           key={"line" + i}
-          color={window.matchMedia("(prefers-color-scheme: dark)") ? "rgb(0,148,236)" : "rgb(77, 77, 77)"}
+          color={"rgb(0,148,236)"}
```

Comment:
For what is this color used? In general no color should be hardcoded maybe there is an opportunity to use a color from our color scheme.

Reply:
Polygon is a map component from Leafletmap and i don't think there is an other way to set the dashed line color.